docs/schema/action-element-actions-elementworkflowinfo-elementaction4.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Action element
+description: Discusses the definition, elements and attributes for Action element which contains the information that is required for Workflow Manager Client.
 manager: laurawi
 ms.date: 3/9/2015
```

Comment:
Your fixes look good! Just don't forget to change the dates on the files you touch to reflect the dates that you made the changes!